share/functions/fish_default_key_bindings.fish

```diff
@@ -76,7 +76,7 @@ function fish_default_key_bindings -d "Default (Emacs-like) key bindings for fis
     bind --preset $argv \ch backward-delete-char
     bind --preset $argv \cp up-or-search
     bind --preset $argv \cn down-or-search
-    bind --preset $argv \cf forward-char
+    bind --preset $argv \cf forward-char # Does not autocomplete entire line by default
```

Comment:
Please make this also accept the entire line by default.

Again: Please don't change the defaults. Accepting the entire line is *a good default*. Accepting single characters isn't of much use.

Reply:
> Accepting single characters isn't of much use.

I strongly disagree, but okay. Default are now the same.